DigitalLearningSolutions.Data/Services/ActionPlanService.cs

```diff
@@ -138,7 +140,9 @@ public async Task AddResourceToActionPlan(int learningResourceReferenceId, int d
             return await MapLearningLogItemsToActionPlanResources(completedLearningLogItems);
         }
 
-        public async Task<ActionPlanResource?> GetActionPlanResource(int learningLogItemId)
+        public async Task<(ActionPlanResource? actionPlanResource, bool sourcedFromFallbackData)> GetActionPlanResource(

```

Comment:
The return value of LearningLogItemsDataService.GetLearningLogItem() is being changed in [747](https://github.com/TechnologyEnhancedLearning/DLSV2/pull/896/files) - `sourcedFromFallbackData` is being inverted and named `apiIsAccessible`. Let's do the same with this return value (the main motivation for this is that we're applying special significance to the return values `(null, false)` and `(null, true)`. In one case the resource is deleted, in the other the API is down and we have no fallback data - that's not very clear if the bool is called `sourcedFromFallbackData`.

Reply:
Should be changed anywhere else where it's applicable too.

Reply:
Hats off